### PR TITLE
fix: Introduced housingCard limit for landing page

### PR DESCRIPTION
**Pull request by dhylan01**

### CODE REVIEW

File: FrontEnd/src/components/HouseCardList/HouseCardList.tsx [hunk 1]
```diff
@@ -8,7 +8,11 @@ import { HouseCardLandLord, HouseCardStudent } from './HouseCard';
 import styles from './HouseCardList.module.scss';
 
 export type PostingType = 'landlord' | 'student';
+interface BrowsingListProps {
+  numListingsShown?: number;
+}
 
+export type displayType = 'layout' | 'all';
```
Comment (Yizong98) [inline FrontEnd/src/components/HouseCardList/HouseCardList.tsx:15]: Why do we need this?

File: FrontEnd/src/components/HouseCardList/index.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
 export { default } from './HouseCardList';
 export * from './HouseCardList';
 
+export * from './';
```
Comment (Yizong98) [inline FrontEnd/src/components/HouseCardList/index.ts:4]: wouldn't this give max stack error? I think you can use `git checkout origin/master src/components/HouseCardList/index.ts` to revert this file to track master

**Review by Yizong98 — COMMENTED** [review]
Just need to clean other files. Otherwise LGTM

File: FrontEnd/src/components/HouseCardList/HouseCardList.tsx (hunk 1 above)
Comment (dhylan01) [reply FrontEnd/src/components/HouseCardList/HouseCardList.tsx:15]: it is used in housePropsUI in the BrowsingList Function component that is exported.

File: FrontEnd/src/components/HouseCardList/index.ts (hunk 2 above)
Comment (dhylan01) [reply FrontEnd/src/components/HouseCardList/index.ts]: i actually did that and it did not chance the file but yeah that file works fin. It was when I changed it that I actually got a max stack error. The change was adding the line: export * from './houseCard';

**Pull request closed without merging**